README.md

```diff
@@ -1,12 +1,12 @@
-<div align="center"><img src="docs/source/logo.svg" width="300"/></div>
-
-# mwclient
+<div align="center">
+    <img src="docs/source/logo.png"/>
```

Comment:
While we're at it, should we also add an `alt` attribute here?

Reply:
Good call, will do.

Reply:
Done.